.coderabbit.yaml

```
@@ -20,6 +20,8 @@ reviews:
   tools:
     languagetool:
         enabled: true
-        level: "picky"
+        level: "default"
 chat:
   auto_reply: true
+knowledge_base:
+  opt_out: true
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)